.travis.yml

```diff
@@ -19,7 +24,6 @@ before_script:
     - sh -c 'if [ "$TRAVIS_PHP_VERSION" != "hhvm-nightly" ]; then echo "extension = memcache.so" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini; fi;'
     - sh -c 'if [ "$TRAVIS_PHP_VERSION" != "hhvm-nightly" ]; then echo "memory_limit = -1" >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini; fi;'
     - sudo locale-gen nl_BE.UTF-8 && sudo update-locale
-    - composer --prefer-source --dev install
     - composer require symfony/symfony:${SYMFONY_VERSION}
     - wget https://phar.phpunit.de/phpunit.phar
```

Comment:
given that Travis updates all its binaries each month, is it really needed to download PHPunit each time?